### ISSUE TITLE
[BUG]: Bruno's display of the response can be modified

### ISSUE THREAD

**Issue by tho-gru-38** [usebruno/bruno#3443]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

While using Bruno I expected that the response frame displays the response as it was received.

Today I learned that the contents of this frame can be modified by JavaScript:
![image](https://github.com/user-attachments/assets/254295f0-761e-4e69-afcf-069a238d3b52)

As you can see above, the JavaScript code changes the displayed contents of the frame (marked green). When you delete the marked line of code in the tests section and rerun the request, the marked property will not be displayed anymore.

My expectation is that Bruno displays the response as it was received. This display should not be modifiable by any JavaScript code. Otherwise I can not see the truth about the response because it might be messed up by some code containing errors.

In summary: res.getBody() should return a deep copy of what is displayed in the response frame.

### .bru file to reproduce the bug

```
meta {
  name: Modify response
  type: http
  seq: 6
}

get {
  url: https://jsonplaceholder.org/users
  body: none
  auth: none
}

tests {
  const jsonResponse = res.getBody();
  bru.setVar("resource_responseBody", jsonResponse);
  
  jsonResponse[0].insertedByScript = "test123-abc-def"; /* */
  
}
```

### Screenshots/Live demo link

.

**Comment by tho-gru-38** [comment]
I found the following workaround:
```
const lodash = require("lodash");
const jsonResponse = lodash.cloneDeep(res.getBody());
bru.setVar("resource_responseBody", jsonResponse);
```

In my opinion this deep copy should be done before returning the reference of the body to make sure that the display of Bruno always reflects the response.

**Comment by sreelakshmi-bruno** [comment]
Hi @tho-gru-38, thanks for reporting this, we'll work this!
